.github/workflows/build.yaml

```diff
@@ -62,7 +63,15 @@ jobs:
       run: make dist
     # Audit all currently installed packages for security vulnerabilities.
     - name: Audit installed packages
-      run: make audit
+      id: audit-packages
+      continue-on-error: true
+      run: make audit > raw-vulnerabilities.txt && tail -n +5 raw-vulnerabilities.txt > vulnerabilities.txt # Saving the command output and tailing from line 5 to avoid having the command on the PR comment
```

Comment:
Have you tried `make -s` here?

Reply:
I didn't realize that! thanks for the suggestion! work's perfectly with the flag 😃